user/obfs4proxy/template.py

```diff
@@ -0,0 +1,16 @@
+pkgname = "obfs4proxy"
+pkgver = "0.0.14"
+pkgrel = 0
+build_style = "go"
+make_build_args = ["./obfs4proxy"]
+hostmakedepends = ["go"]
+pkgdesc = "Circumvents censorship by transforming client-to-bridge Tor traffic"
+license = "GPL-3.0-or-later"
```

Comment:
there's also https://github.com/Yawning/obfs4/blob/master/LICENSE? but i haven't looked much into the licensing situation of this